libs/core/src/lib/grid-list/components/grid-list-item/grid-list-item.component.html

```diff
@@ -143,6 +144,8 @@
         [id]="id + '-radio'"
         [name]="'fd-grid-list-item-toolbar-' + selectionMode"
         [attr.aria-label]="ariaLabel"
+        [attr.aria-posinset]="posInSet"
+        [attr.aria-setsize]="_totalItemsCount"
```

Comment:
Please check [this documentation](https://www.digitala11y.com/aria-posinset-properties/) and explain why these attributes are added.
I think this is the case for Grid List: "If all items in a set are present in the document structure, it is not necessary to set this attribute, as the user agent can automatically calculate the set size and position for each item."

Reply:
It looks like screen reader fails to automatically calculate the set size here. This is a fix to ISSUE 11 from the attached doc.

Reply:
But the is it really an a11y issue or a screen reader issue? I think we need to comply with the a11y official documentation and guidelines, not to a certain version of a screen reader. 
We are having a meeting with the a11y team next week. Blocking this PR for now. Sorry about that.

Reply:
Hello Colleagues, 

The radio button count is reading precise now.